cmd/config.toml.example

```diff
@@ -2,6 +2,7 @@
 NetworkAddress = <a network address, Eg: http://127.0.0.1:8545 for ganache>
 BridgeAddress = <the eth address for the bridge contract>
 PrivateKey = <the private key of the relayer eth wallet>
+GasLimit = 400000
```

Comment:
Based on the comment I left here https://github.com/ElrondNetwork/elrond-eth-bridge/issues/45#issuecomment-926359776 right now a sensible default is 500000